.github/workflows/build-streams.yaml

```diff
@@ -7,25 +7,122 @@ on:
       - release-sentry-streams/**
 
 jobs:
-  dist:
-    name: Create wheel and source distribution for sentry-streams
-    timeout-minutes: 10
+  linux:
+    permissions: read-all
+    timeout-minutes: 5
+    runs-on: ${{ matrix.platform.runner }}
+    strategy:
+      matrix:
+        platform:
+          - runner: ubuntu-latest
+            target: x86_64
+          - runner: ubuntu-latest-arm
+            target: aarch64
+
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-python@v5
+        with:
+          python-version: 3.11
+      - name: Build wheels
+        uses: PyO3/maturin-action@v1
+        with:
+          target: ${{ matrix.platform.target }}
+          args: --release --out dist --find-interpreter
+          sccache: ${{ !startsWith(github.ref, 'refs/tags/') }}
+          manylinux: auto
```

Comment:
I don't think we want to set `manylinux: auto` here, better to be specific and pin this since we're publishing a wheel here and don't know if we want to trust auto here

Reply:
done. set to 2014